addons/website_sale/static/src/js/website_sale.js

```diff
@@ -8,7 +8,7 @@ var _t = core._t;
 var timeout;
 
 publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
-    selector: '#top_menu a[href$="/shop/cart"]',
+    selector: '#top a[href$="/shop/cart"]',
```

Comment:
Actually, sorry to be annoying but maybe do this: `'#top a[href$="/shop/cart"], #top_menu a[href$="/shop/cart"]'`

That way, we don't take the risk of breaking something in stable. I could very well imagine someone changing the id of the header as a custo :wink: Just indicate in a comment that the second part can be removed in master :+1:

Reply:
No problem, I actually had this fix in mind for the case you would say it's not ok for stable haha